project-fortis-pipeline/docs/development-library.md

```diff
@@ -0,0 +1,154 @@
+# Commonly Used Libraries

```

Comment:
This only lists the Node libraries, but we also have Bash libraries, Scala libraries, etc.